spec/controllers/follows_controller_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
```

Comment:
I thought we were just doing feature tests these days, not the others? Or was it just view tests that we decided to stop doing?